dashboard/app/mailers/pd/workshop_mailer.rb

```diff
@@ -288,4 +289,40 @@ def detail_change_notification_subject(workshop)
       'Details for your upcoming Code.org workshop have changed'
     end
   end
+
+  def get_contact_text_for_teacher_follow_up(workshop)
```

Comment:
Since there is no test for this function, could you add comments to explain what the outputs will be.

Reply:
And thanks for moving the code into a function. It will be easier to generalize it and add tests if we have to implement the same logic for other emails.

Reply:
added comment and tests